doc/appendix-de-kassensichv/cash-register-integration/receipt-sequences-creation.md

```diff
@@ -0,0 +1,174 @@
+---
+slug: /poscreators/middleware-doc/germany/receipt-sequences-creation
+title: Receipt sequences creation
+---
+
+# Receipt sequences creation
+
+In the chapter [Single receipt creation](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/germany/single-receipt-creation), the creation of single receipts using either implicit and/or explicit flow has been described.
+
+In this chapter, we will describe how to connect those single receipts to receipt sequences to integrate complex business cases.
+
+## Referencing previous receipts within a queue
```

Comment:
maybe we should think about a another wordin see comment line 33

Reply:
we'll use "actions"